### PR TITLE
English translation - First Steps

### PR DESCRIPTION
**Pull request by cconard96**
Goals:
- Translate to English to prepare for localization via Transifex or another platform
- Minor rewording of some sections to so it is easier for new GLPI users (Not just GLPI developers and long-time users) to understand

First pass of translation is mostly automated through Chrome's Google Translate feature so I understand it (I can't read French). Second pass will be me rewording sections so a native English speaker understands it. I will try to keep it closer to European English as the base language of GLPI is en_GB and not en_US.

### CODE REVIEW

**Comment by cconard96** [comment]
The entire First Steps/Getting Started section of documentation has been translated and ready for review.

File: source/first-steps/general.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* Edge
+* Firefox (including 2 latests ESR version)
+* Chrome
+GLPI also works on mobile; it is generally compatible with the mobile versions of the supported browsers.
+
+How to connect
+--------------------
+
+Open your browser and go to the GLPI homepage `https://{glpi_address}/ <https://{glpi_address}>`_).
```
Comment (orthagh) [inline source/first-steps/general.rst:21]: There is a trailing parenthensis here, desired ?

**Comment by trasher** [comment]
Hi,

We've setup build on pull requests on readthedocs; result for current PR is visible at:
https://glpi-user-documentation--37.org.readthedocs.build/fr/37/

File: source/first-steps/interfaces.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+   Offers the same information about tickets and issues as the personal view, but using the groups to which it belongs as a reference. Depending on their profile, they may or may not have access to the tickets for which their groups are requesting.
+
+* *Global view*
+
+   Offers statistics on tickets and issues according to their status and on contracts according to their expiration date. It also offers the view of the latest additions of objects in GLPI. Depending on the configuration used, new tickets can also be presented in this tab.
+
+* *RSS feed*
+
+   Proposes the content of defined RSS feeds [Gérer les flux RSS](06_Module_Outils/04_Flux_RSS.rst "Les flux RSS se gèrent depuis le menu Outils > Flux RSS")).
```
Comment (orthagh) [inline source/first-steps/interfaces.rst:41]: I don't know if it's a typo or a cons of EN translation, but the link here doesn't work on rtfm build:

File: source/first-steps/interfaces.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.. note::
+
+   For Personal and Group views, only tables with information are displayed.
+
+.. note::
+
+   Two security messages may be displayed when you log in for the first time after completing the installation procedure.
+
+   The first asks you to change the passwords of the accounts created by default in GLPI, the second asks you to delete the directory ``glpi/install``.
```
Comment (orthagh) [inline source/first-steps/interfaces.rst:51]: to note for future works (as the current pr is for translation), we may have additionnal messages here (migrations required  mainly)

File: source/first-steps/interfaces.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+   Offers the same information about tickets and issues as the personal view, but using the groups to which it belongs as a reference. Depending on their profile, they may or may not have access to the tickets for which their groups are requesting.
+
+* *Global view*
+
+   Offers statistics on tickets and issues according to their status and on contracts according to their expiration date. It also offers the view of the latest additions of objects in GLPI. Depending on the configuration used, new tickets can also be presented in this tab.
+
+* *RSS feed*
+
+   Proposes the content of defined RSS feeds [Gérer les flux RSS](06_Module_Outils/04_Flux_RSS.rst "Les flux RSS se gèrent depuis le menu Outils > Flux RSS")).
```
Comment (trasher) [inline source/first-steps/interfaces.rst:41]: Should be something like (not tested at all):
```suggestion
   Proposes the content of defined RSS feeds :doc:`manage RSS feeds <modules/outils/fulxrss>` "Les flux RSS se gèrent depuis le menu Outils > Flux RSS")).
```

File: source/first-steps/interfaces.rst (hunk 2 above)
Comment (cconard96) [reply source/first-steps/interfaces.rst]: It was a hold-over from the original FR documentation. There are a few dead links in the docs that will need converted.

Comment (orthagh) [reply source/first-steps/interfaces.rst]: ok

**Review by orthagh — APPROVED** [review]

**Pull request merged**